.cirrus.yml

```diff
@@ -134,41 +134,48 @@ Bids_validate_task:
 
 Integration_windows_task:
   windows_container:
-    matrix:
-      - dockerfile: .ci/w_python_39
-      - dockerfile: .ci/w_python_38
-      - dockerfile: .ci/w_python_37
-      - dockerfile: .ci/w_python_36
+    dockerfile: .ci/w_miniconda
     cpu: 1
+  env:
+    matrix:
+      PYTHON: 3.9
+      PYTHON: 3.8
+      PYTHON: 3.7
+      PYTHON: 3.6
   environment_cache:
     folder: shared39
     fingerprint_script: echo 362
     reupload_on_changes: true
     populate_script:
-      - pip install msvc-runtime
-      - pip install Cython
+      - conda create -yq --prefix  C:\Users\ContainerAdministrator\AppData\Local\Temp\cirrus-ci-build\conda-env python=%PYTHON%
+      - activate C:\Users\ContainerAdministrator\AppData\Local\Temp\cirrus-ci-build\conda-env
       - pip install -e .[test,doc]
   Running_integration_script:
     - choco install make -y --no-progress
+    - activate C:\Users\ContainerAdministrator\AppData\Local\Temp\cirrus-ci-build\conda-env
     - pip install msvc-runtime
     - make integration
 
 Unittest_windows_task:
   windows_container:
-    matrix:
-      - dockerfile: .ci/w_python_39
-      - dockerfile: .ci/w_python_38
-      - dockerfile: .ci/w_python_37
-      - dockerfile: .ci/w_python_36
+    dockerfile: .ci/w_miniconda
     cpu: 1
+  env:
+    matrix:
+      PYTHON: 3.9
+      PYTHON: 3.8
+      PYTHON: 3.7
+      PYTHON: 3.6
   environment_cache:
     folder: shared39
     fingerprint_script: echo 362
     reupload_on_changes: true
     populate_script:
-      - pip install msvc-runtime
+      - conda create -yq --prefix  C:\Users\ContainerAdministrator\AppData\Local\Temp\cirrus-ci-build\conda-env python=%PYTHON%
```

Comment:
just want to make sure I understand what's happening here: this will build environments with each of the Python versions mentioned above, yes?

Reply:
Exactly, the matrix setup above creates a set of jobs that are identical except for the variable %PYTHON%, which indicates the desired python version for each job

Reply:
I wanted to to this change to equivalate the windows and linux jobs. Now whenever we have a new python version we only need to add a line to each task ( if we don't run in installation trouble later)

Reply:
👍 sounds good!